addons/controllers/testdata/dependency/crd/vmoperator.vmware.com_virtualmachines.yaml

```diff
@@ -0,0 +1,535 @@
+
```

Comment:
`vm-operator-api` doesn't include CRD definition yaml files, so we need to manually import.

Reply:
What do you need